src/common/dataHooks/useMediaSearch.js

```diff
@@ -1,14 +1,27 @@
 import { useEffect, useState } from 'react'
-import { useSearchParams } from 'react-router-dom'
+import { useParams, useSearchParams } from 'react-router-dom'
 import PropTypes from 'prop-types'
+import { useInfiniteQuery } from '@tanstack/react-query'
 
 // FPCC
-import useSearchLoader from 'common/dataHooks/useSearchLoader'
-import useIntersectionObserver from 'common/hooks/useIntersectionObserver'
-import { AUDIO, IMAGE, VIDEO, TYPES, SORT, SORT_CREATED_DESC } from 'common/constants'
+import useLoader from 'common/hooks/useLoader'
+import api from 'services/api'
+import { searchResponseAdaptor } from 'common/dataAdaptors'
+import {
+  AUDIO,
+  IMAGE,
+  VIDEO,
+  TYPES,
+  SEARCH,
+  SHARED_MEDIA,
+  SORT,
+  SORT_CREATED_DESC,
+} from 'common/constants'
 import { getPathForMediaType } from 'common/utils/mediaHelpers'
 
-function useMediaSearch({ type }) {
+function useMediaSearch({ type, library }) {
+  const { sitename } = useParams()
+  const siteToSearch = library === SHARED_MEDIA ? SHARED_MEDIA : sitename
```

Comment:
Did you see the tech design notes for this search? https://firstvoices.atlassian.net/wiki/spaces/FIR/pages/425132035/Shared+Image+Library+requirements

To avoid having to hardcode a site name, there is a parameter in the cross-site search for finding shared media. The intention was to be able to support multiple shared media sites if we wanted to.

Reply:
Was the tech design implemented on the BE according to the gold stars?